v2.1/performance-benchmarking.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Performance Benchmarking
+summary: Learn how to benchmark CockroachDB against TPC-C.
+toc: false
+---
+
+This page walks you through [TPC-C](http://www.tpc.org/tpcc/) performance benchmarking on CockroachDB. It measures `tpmC` (new order transactions/minute) on two TPC-C datasets:
+
+- 1,000 warehouses (for a total dataset size of 200GB) on 3 nodes
+- 10,000 warehouses (for a total dataset size of 2TB) on 30 nodes _(Coming soon)_
+
+These two points on the spectrum show how CockroachDB scales from modest-sized production workloads to larger-scale deployments.
+
+<!--This demonstrates how CockroachDB achieves high OLTP performance of over 128,000 tpmC on a TPC-C dataset over 2TB in size.-->
+
+<div id="toc"></div>
+
+## Benchmark a small cluster
+
+### Step 1. Start a cluster on Google Cloud Platform GCE
+
+Follow steps 1-7 in the [GCE tutorial to deploy a 3-node CockroachDB cluster on Google Cloud](deploy-cockroachdb-on-google-cloud-platform.html), with the following changes:
+
+- For the 3 CockroachDB nodes, use the `n1-highcpu-16` VMs with [Local SSD](https://cloud.google.com/compute/docs/disks/local-ssd).
```

Comment:
nits:
- remove `the` before `n1-highcpu-16`
- Change `Local SSD` to `Local SSD storage`.